source/tutorials/first-steps/reproducible-scripts.md

```diff
@@ -92,4 +92,9 @@ Run the script:
 
 - {ref}`reading-nix-language` to learn about the Nix language, which is used to declare packages and configurations.
 - {ref}`declarative-reproducible-envs` to create reproducible shell environments with a declarative configuration file.
-- [Garbage Collection](https://nix.dev/manual/nix/2.18/package-management/garbage-collection.html) – free up storage used by the programs made available through Nix
+
+If you're done trying out Nix for now, you can free up storage used by the programs made available through Nix:
+
+```shell-session
+$ nix-collect-garbage
+```
```

Comment:
Feels off to have a mini-tutorial in next steps section. What's the problem with the previous text?

Reply:
It points to a fairly useless manual page, and I have work piled up to fix that. But there's nothing you  need to know to run the command. I can leave it though, it will get better eventually.